src/OpenTelemetry.Contrib.Instrumentation.Runtime/RuntimeMetrics.cs

```diff
@@ -100,5 +107,21 @@ private static double GetFragmentation()
             return gcInfo.HeapSizeBytes != 0 ? gcInfo.FragmentedBytes * 100d / gcInfo.HeapSizeBytes : 0;
         }
 #endif
+
+        private double CalculateCpuUsage()
+        {
+            var cpuUsage = 0.0;
+
+            var endTime = Process.GetCurrentProcess().TotalProcessorTime.TotalMilliseconds;
```

Comment:
won't ObservableCounter work better? We can simply pass `Process.GetCurrentProcess().TotalProcessorTime.TotalMilliseconds` value to it. (and let SDK/downstream handle calculation?

Reply:
@reyang @noahfalk to share thoughts here.

Reply:
https://github.com/open-telemetry/opentelemetry-dotnet-contrib/pull/207#discussion_r816413966

Reply:
As mentioned in the [semantic conventions](https://github.com/open-telemetry/opentelemetry-specification/blob/main/specification/metrics/semantic_conventions/process-metrics.md#process) we should (also) provide a "process.cpu.time" with the implementation of `Process.GetCurrentProcess().TotalProcessorTime.TotalMilliseconds`

Reply:
I'd suggest we have process.cpu.time be a replacement for this rather than in addition to this. However if we do decide have a runtime metric ObservableCounter still seems like the prefered way to do it. 

I could be messing up my math, but the current code doesn't appear to calculate the intended result. For example assume we have a 4 core machine, measurements are taken once a minute and the process is using all available CPU. At T=1 min TotalMilliseconds returns 4*60*1000=240_000. At T=2 min TotalMilliseconds returns 2*4*60*1000=480_000. The metric computes (480K-240K)/(100*4) = 600%

Reply:
I've changed to `process.cpu.time`.